dali/test/python/reader/test_webdataset_requirements.py

```diff
@@ -274,6 +274,150 @@ def test_pax_format():
         )
 
 
+def test_case_sensitive_container_format():
+    global test_batch_size
```

Comment:
I can see this is used in other test cases in this file, but is it needed? It seems to only unnecessary circumvent Python protection against accidental modification of variables in global scope.

Reply:
Apparently not. Removed.